finalPrep/modZeroHero.js

```diff
@@ -74,11 +142,46 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 
 // - Create 2 instances of your SuperHero class
 
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+  }
+    static archNemesis = "The Syntax Error"
+    static powerLevel = 100
+    static energyLevel = 50
+
+    sayName() {
+      console.log('It is me, ' + this.name + '!');
+    }
+    maximizeEnergy() {
+      SuperHero.energyLevel = 1000;
```

Comment:
It isn't just any `SuperHero` you want to apply that energy level to.... it is `this` superhero. `this` is a tough topic you will explore a lot more at Turing, but for now, get used to using that term a lot with classes when you are changing values of attributes.